### DISCUSSION TITLE
Possesion mod like controlcompany?

### DISCUSSION THREAD

**Discussion by ivjvon** [winstxnhdw/lc-hax#295] [Q&A]
Is the possession mod the same as controlcompany? Or is it only movement, how do you attack? I tried controlling a bracken and all it does is nothing, mouse1 & 2 are empty

**Comment by joep26020** [comment]
> Is the possession mod the same as controlcompany? Or is it only movement, how do you attack? I tried controlling a bracken and all it does is nothing, mouse1 & 2 are empty

1. have you tried to see of this is a you issue
2. have you tried to see if others have had the same issue
3. have you tried to solve the issue
4. have you reinstalled the software

**Comment by winstxnhdw** [comment]
Tracked in #250
